src/wallet/utils/validChainId.ts

```diff
@@ -0,0 +1,15 @@
+import { validChainIdOptions, validChainIdResponse } from '../types';
+import { baseSepolia, base } from 'viem/chains';
+
+/**
+ * validChainId
+ *  - Checks if the paymaster operations chain id is valid
+ *  - Only allows the base and baseSepolia chain ids
+ */
+export function validChainId({ chainId }: validChainIdOptions): validChainIdResponse {
+  if (chainId !== baseSepolia.id && chainId !== base.id) {
+    return { isValid: false, error: 'Invalid chain id', code: 1 };
```

Comment:
@0xAlec @cpcramer  there is something here we should probably standardize on how `code` should be interepreted.

I can easily imagine how the `code` has to be unique across all OnchainKit. So if that is the case `code` should be a `string`. And maybe somehting like `W_ERR_1` where the first character is about the Kit prefix, and than there is a number.

Or how others do?